development/kops/install_requirements.sh

```diff
@@ -69,7 +69,7 @@ then
 fi
 
 echo "$(which kops): $(kops version)"
-echo "$(which kubectl): $(kubectl version --client=true --short)"
+echo "$(which kubectl): $(kubectl version --client=true)"
```

Comment:
This should check which version `$(RELEASE_BRANCH)` is being tested. Only `1-28` deprecates `--short`

Reply:
like this?

```
if [[ "${RELEASE_BRANCH}" == "1-28" ]]; then
    echo "$(which kubectl): $(kubectl version --client=true)"
else
    echo "$(which kubectl): $(kubectl version --client=true --short)"
fi
```

if yes, what would happen with the next versions?

Reply:
The deprecation occurred with 1.28+ I would recommend something like this. It will use the `--short` flag on all versions that still use it without needing to update with future versions.
```
if [[ $test == "1-23" || $test == "1-24" || $test == "1-25" || $test == "1-26" || $test == "1-27" ]]; then
  echo "kubectl version --client=true --short"
else
  echo "kubectl version --client=true"
fi
```